Framework/Algorithms/src/Q1DWeighted.cpp

```diff
@@ -91,6 +95,12 @@ void Q1DWeighted::init() {
 
   declareProperty("AccountForGravity", false,
                   "Take the nominal gravity drop into account.");
+
+  declareProperty(
+      std::make_unique<WorkspaceProperty<ITableWorkspace>>(
+          "ShapeTable", "", Direction::Input, PropertyMode::Optional),
+      "Table containing the shapes for integration. Only sectors supported. If "
```

Comment:
In the short documentations, the docstring is truncated at first `.`.
So you should avoid using them in the middle in general.
Particularly here I'd suggest:
```suggestion
      "Table workspace containing the shapes (sectors only) drawn in the instrument viewer; if specified, the wedges properties defined above are not taken into account."
```